examples/cookbook/effects/expandable_fab/lib/excerpt1.dart

```diff
@@ -15,10 +15,10 @@ class ExpandableFab extends StatefulWidget {
   final List<Widget> children;
 
   @override
-  _ExpandableFabState createState() => _ExpandableFabState();
+  ExpandableFabState createState() => ExpandableFabState();
 }
 
-class _ExpandableFabState extends State<ExpandableFab> {
+class ExpandableFabState extends State<ExpandableFab> {
```

Comment:
I can see how you got here, but it's the wrong fix. You are exposing the `State` object as part of the externalised API, when it is a hidden implementation detail by design. Please revert this line back to `class _ExpandableFabState ...`